src/System.Private.Xml/src/System/Xml/Core/HtmlEncodedRawTextWriter.cs

```diff
@@ -1,18 +1,8 @@
+﻿
```

Comment:
1. Please try to get rid of the extra newline
2. Please add some comment that this file is auto-generated so it's less likely to be edited by hand